code/modules/jobs/job_types/roguetown/Inquisition/puritan.dm

```diff
@@ -25,9 +26,163 @@
+	neck = /obj/item/clothing/neck/roguetown/psicross/silver
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+	pants = /obj/item/clothing/under/roguetown/tights/black
+	cloak = /obj/item/clothing/cloak/cape/puritan
+	backr = /obj/item/storage/backpack/rogue/satchel/black
+	beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
+	head = /obj/item/clothing/head/roguetown/puritan
+	gloves = /obj/item/clothing/gloves/roguetown/leather
+	beltl = /obj/item/rogueweapon/sword/rapier
+	backpack_contents = list(/obj/item/storage/keyring/puritan = 1, /obj/item/lockpickring/mundane = 1, /obj/item/rogueweapon/huntingknife/idagger/silver)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/knives, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 3, TRUE) //Keeping it up with Confessor
+		H.mind.adjust_skillrank(/datum/skill/misc/lockpicking, 5, TRUE) ///Ortho Confessor gets master in every stealth skill. Insane. 
+		H.mind.adjust_skillrank(/datum/skill/misc/tracking, 5, TRUE) //To track down heretics to their evil lairs... 
+		H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 5, TRUE) //Always thought it was crazy they got listening devices but NO means to effectively sneak them into places.
+		if(H.age == AGE_MIDDLEAGED)
+			H.mind.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+			H.change_stat("perception", 1)
+		if(H.age == AGE_OLD)
+			H.mind.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 1, TRUE)
+			H.change_stat("perception", 2)
+			H.change_stat("strength", 2) //to offset the age
+		H.change_stat("strength", 1) ///They deserve the whips too actually
+		H.change_stat("endurance", 3) ///Who the fuck gave Confessor 3 Endurance lmao
+		H.change_stat("perception", 3)
+		H.change_stat("speed", 3) //Rogues get 3 fucking speed and spymaster hands get 4, they deserve this. Confessor does too wtf
+		H.change_stat("intelligence", 3)
+	H.verbs |= /mob/living/carbon/human/proc/faith_test
+	H.verbs |= /mob/living/carbon/human/proc/torture_victim
+	ADD_TRAIT(H, TRAIT_NOSEGRAB, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_STEELHEARTED, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_DODGEEXPERT, TRAIT_GENERIC)
```

Comment:
dodge expert and 4 in swords is the identity of an entire adventurer class, the rogue duelist. this class is like three rogue duelists in a trenchcoat. i know we hate adventurers but maybe we shouldn't step all over their toes.

Reply:
Just because another role has high swords and dodge expert doesn't mean that this is an attempt to mimic the Rogue Duelist. Two roles can have similar skill spreads. Inquisitor should be more powerful than Adventurers.

Reply:
fair tbh i think this one was a bit specious